list.go

```diff
@@ -58,6 +73,14 @@ func List(c *cli.Context) error {
 		})
 	}, 0)
 
+	if c.Bool("priority") == true {
+		// sort output by priority
+		sortItems(&itemList, 1)
+	} else {
+		// sort output by task id, no need to call as items returned by API are already sorted by task id
```

Comment:
Please remove this `else` block.
And move this comment to above the if statement.

Reply:
`else` block has been removed.